rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -630,20 +653,15 @@ protected Integer commitFederation(boolean dryRun, Keccak256 hash, BridgeEventLo
         oldFederationUTXOs.clear();
         oldFederationUTXOs.addAll(utxosToMove);
 
-        // Network parameters for the new federation are taken from the bridge constants.
-        // Creation time is the block's timestamp.
         Instant creationTime = Instant.ofEpochMilli(rskExecutionBlock.getTimestamp());
+        long rskExecutionBlockNumber = rskExecutionBlock.getNumber();
```

Comment:
Maybe call it federationCreationBlockNumber? A bit more explanatory I think

Reply:
done